requirements.txt

```diff
@@ -2,4 +2,3 @@ Django>=4.2
 django-environ
 bumpversion
 mock
-sphinx_rtd_theme==3.0.2
```

Comment:
Is there a dependency we now need to have for mkdocs? eg to preview builds in our local tools?